scripts/sanity_check.sh

```diff
@@ -130,15 +130,18 @@ check_systemctl_services() {
     check_each_system_service firemon "running"
     check_each_system_service firekick "dead"
     check_each_system_service redis-server "running"
-    check_each_system_service openvpn@server "running"
-    check_each_system_service watchdog "running"
     check_each_system_service brofish "running"
     check_each_system_service firewalla "dead"
     check_each_system_service fireupgrade "dead"
     check_each_system_service fireboot "dead"
 
+    vpn_state=$(redis-cli hget policy:system vpn | jq .state)
```

Comment:
just be aware that jq is not available in old Red/Blue

Reply:
replaced with `fgrep`